frontend/lib/app.dart

```diff
@@ -73,24 +75,24 @@ class App extends StatelessWidget {
     final mapStyleUrl = settings.enableStaging
         ? buildConfig.mapStyleUrl.staging
         : isProduction()
-            ? buildConfig.mapStyleUrl.production
-            : isLocal()
-                ? buildConfig.mapStyleUrl.local
-                : buildConfig.mapStyleUrl.showcase;
+        ? buildConfig.mapStyleUrl.production
+        : isLocal()
+        ? buildConfig.mapStyleUrl.local
+        : buildConfig.mapStyleUrl.showcase;
```

Comment:
The indentation here helps understanding the code. I would either add brackets or replace the whole construct with a `switch` expression:

```dart
    final mapStyleUrl = switch (appEnvironment) {
      'showcase' => buildConfig.mapStyleUrl.showcase,
      'production' => buildConfig.mapStyleUrl.production,
      _ => buildConfig.mapStyleUrl.local,
    };
```

Reply:
We can not determine staging environment with app environment, since we don't distinguish the app build.
Thats why such an easy switch case won't work because we don't get staging from environment

Reply:
we could use this

```
    final mapStyleUrl = settings.enableStaging
        ? buildConfig.mapStyleUrl.staging
        : switch (appEnvironment) {
      'production' => buildConfig.mapStyleUrl.production,
      'local' => buildConfig.mapStyleUrl.local,
      _ => buildConfig.mapStyleUrl.showcase,
    };
```